recipes/azure-storage/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = "0.20.3" %}
+
+package:
+  name: azure-storage
+  version: {{ version }}
+
+source:
+  fn: azure-storage-{{ version }}.zip
+  url: https://pypi.python.org/packages/9b/ce/60a1173299afc98814e582c419cf84b358c083ad86fb21bafd801548ca5b/azure-storage-{{ version }}.zip
```

Comment:
Please change to `https://pypi.io/packages/source/a/azure-storage/azure-storage-{{ version }}.zip`.